pkg/model/serializer.go

```diff
@@ -37,6 +38,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+
+	// ErrDevModeNotValid is raised when development mode in manifest is not 'sync' nor 'hybrid'
```

Comment:
```suggestion
	// errDevModeNotValid is raised when development mode in manifest is not 'sync' nor 'hybrid'
```

or consider removing it

Reply:
🙏 e3485e01ea5c2cab6bd82bacf0b5ba733ba5b1c7 . I keep it because I think it is not superfluous to have a description of the error.